include/language/en_us.lang.php

```diff
@@ -3494,8 +3494,8 @@
 
 //SecurityGroups
 $app_list_strings['moduleList']['SecurityGroups'] = 'Security Suite Management';
-$app_strings['LBL_LOGIN_AS'] = 'Login as ';
-$app_strings['LBL_LOGOUT_AS'] = 'Logout as ';
+$app_strings['LBL_LOGIN_AS'] = 'Login as';
+$app_strings['LBL_LOGOUT_AS'] = 'Logout as';
```

Comment:
These lines are also unused, why you didn’t remove them?

Reply:
I have taken these out :+1:  https://github.com/salesagility/SuiteCRM/pull/8065